freeipa.spec.in

```diff
@@ -520,6 +520,22 @@ If your network uses IPA for authentication, this package should be
 installed on every client machine.
 This package provides command-line tools for IPA administrators.
 
+%package client-samba
+Summary: Tools to configure Samba on IPA client
+Group: System Environment/Base
+Requires: %{name}-client = %{version}-%{release}
+Requires: python3-samba
+Requires: samba-client
+Requires: samba-winbind
+Requires: samba-common-tools
+Requires: samba
+Requires: sssd-winbind-idmap
+Requires: tdb-tools
+Requires: cifs-utils
+
+%description client-samba
+This package provides command-line tools to deploy Samba domain member
+on the machine enrolled into a FreeIPA environment
```

Comment:
Is this packaged separately to enforce the idea that other work is needed to setup a Samba share (setup Trust)? Is there any other reason not to just include this alongside ipa-client-install?

Reply:
If you would have it in freeipa-client package, then it would pull in Samba packages on every deployment. I don't think this is really intended.

Reply:
Heh, no, I didn't read this carefully enough! A separate package it is :-)